src/config/settings.py

```diff
@@ -19,6 +18,9 @@
     map(str.strip, env.list("CSRF_TRUSTED_ORIGINS", default=["https://127.0.0.1", "https://localhost"]))
 )
 
+
+PERSISTANCE_PATH = "persistance_data/persistance_file"
```

Comment:
```suggestion
PERSISTANCE_DIR = BASE_DIR / "persistance_data"
PERSISTANCE_PATH = PERSISTANCE_DIR / "persistance_file"
```

Давай сделаем по примеру как сделано с каталогом для логов. Каталог для файла определим отдельно и тут же сделаем проверку на наличие нужного каталога для файла. Тогда код из файла `bot.py` пропадет.